go/coding_standard.md

```diff
@@ -31,18 +31,32 @@ Please add items here, or modify them as appropriate. We have git versioning, do
  * Many comments are not sentences, these should begin with a lower case letter and end without a period.
  * The first letter of sentences in comments are capitalized and ends with a period.
  
-## Git
-
- * Production repos never push to `master` or `develop` branches except the push-master.
- * `develop` should always pass tests.
- * Always start work with a "WIP: \*" (work-in-progress) pull-request, on a branch ideally named "feature/\*" or "hotfix/\*". You're encouraged to fork the repo and add it as a remote in .git/config (so you can push to your fork, but still use the same import paths to the original project).
- * When complete, pull requests should be opened to the `develop` branch.
- * Do not merge PRs that aren't final -- e.g. no code dead code or code that won't be used should be committed.
- * Squash and merge to keep the history clean.
- * Merging from `develop` to `master` happens during a release.
- * All branches should be in lowercase, underscores are okay in branch names
- * All pull requests should include any relevant additions to the `CHANGELOG.md`, If PR is to the develop branch changes
- can be included under an `unreleased changes` header at the top of the document.
+## Github
+
+Developement should following the regimented strucutre for branches and pull requests outlined herein. 
+For further inspiration checkout https://nvie.com/posts/a-successful-git-branching-model/.
+
+ * Production branches
+   * Production repos never push to `master` or `develop` branches except the push-master.
+   * `develop` should always pass tests.
+   * `develop` branch should have an on going open PR to `master`, high level comments can be made here
```

Comment:
What's the benefit here? It assumes there is a clear idea of a the next release a priori.

Reply:
Do you think it's a bad idea to track high level comments here? Care to expand a bit on your thinking?

Reply:
My point is that the PR on-going does add much value, create one at the time of the release and take the contents from the PRs merged into `develop` as basis for the release, no need to have two places to track, kept up-to-date.

Reply:
Cool - removing